src/main/java/Task.java

```diff
@@ -0,0 +1,35 @@
+public class Task {
+    private String description;
+    private boolean done;
```

Comment:
booleans variables should be named to sound like booleans
```suggestion
    private boolean isDone;
```